contrib/neovide/patches/fix-skia-bindings.patch

```diff
@@ -0,0 +1,28 @@
+--- a/neovide/vendor/skia-bindings/build_support/skia_bindgen.rs
++++ b/neovide/vendor/skia-bindings/build_support/skia_bindgen.rs
+@@ -805,15 +805,18 @@ pub(crate) mod definitions {
+     }
+ 
+     fn ninja_files_for_features(features: &features::Features) -> Vec<PathBuf> {
++        use crate::skia::env;
+         let mut files = vec!["obj/skia.ninja".into()];
+         if features.text_layout {
+             files.extend(vec![
+                 "obj/modules/skshaper/skshaper.ninja".into(),
+                 "obj/modules/skparagraph/skparagraph.ninja".into(),
+                 // shaper.cpp includes SkLoadICU.h
+-                "obj/third_party/icu/icu.ninja".into(),
+                 "obj/modules/skunicode/skunicode.ninja".into(),
+             ]);
++            if !env::use_system_libraries() {
++                files.push("obj/third_party/icu/icu.ninja".into());
++            }
+         }
+         if features.svg {
+             files.push("obj/modules/svg/svg.ninja".into());
+--- a/neovide/vendor/skia-bindings/.cargo-checksum.json
```

Comment:
unneeded/unwanted, use cargo.clear_vendor_checksums

Reply:
also i'm about 95% sure you'll need to further patch skia (see ppc-musttail.patch in firefox/thunderbird, similar in libreoffice and chromium)

Reply:
added ppc-musttail.patch from firefox